internal/provider/resource_docker_service_test.go

```diff
@@ -437,7 +437,7 @@ func TestAccDockerService_fullSpec(t *testing.T) {
 			s.Spec.TaskTemplate.ContainerSpec.Secrets[0].File.UID != "0" ||
 			s.Spec.TaskTemplate.ContainerSpec.Secrets[0].File.GID != "0" ||
 			// nolint: staticcheck
-			s.Spec.TaskTemplate.ContainerSpec.Secrets[0].File.Mode != os.FileMode(777) {
```

Comment:
I quickly looked into the code, and it seems like the whole parsing of the FileMode option is weird. I would ask you to leave this out of this PR and create an issue for that, so we can tackle this at a later point

Reply:
Okay! One sec...

Reply:
Fix removed from this patch